tools/privacyidea-token-janitor

```diff
@@ -248,6 +254,10 @@ def build_tokenvalue_filter(key,
     tvfilter = []
     if tokeninfo_value:
         tvfilter.append(_compare_regex_or_equal(key, tokeninfo_value))
+    if has_not_tokeninfo_key:
+        tvfilter.append(has_not_tokeninfo_key)
+    if has_tokeninfo_key:
+        tvfilter.append(has_tokeninfo_key)
```

Comment:
this makes no sense, we would need a compatible database query here.

Reply:
All changes fixed in commit 7d081601e5905194024ef14f932dd101012fe3a3.